otherlibs/win32unix/symlink.c

```diff
@@ -28,32 +28,80 @@
 #include <caml/osdeps.h>
 #include "unixsupport.h"
 
+#ifndef SYMBOLIC_LINK_FLAG_ALLOW_UNPRIVILEGED_CREATE
+#define SYMBOLIC_LINK_FLAG_ALLOW_UNPRIVILEGED_CREATE (0x2)
+#endif
+
 typedef BOOLEAN (WINAPI *LPFN_CREATESYMBOLICLINK) (LPWSTR, LPWSTR, DWORD);
 
+static BOOL is_initialized = FALSE;
 static LPFN_CREATESYMBOLICLINK pCreateSymbolicLink = NULL;
-static int no_symlink = 0;
+static BOOL no_symlinks_available = FALSE;
```

Comment:
I would use `symlink_available` to avoid the double negative.

Reply:
Actually, and apologies for the back-and-forth, this change is just a renaming, is that right? In that case, it would be better to leave it as before (`no_symlink`), as to reduce the diff to the absolute minimum (easier to review).

Reply:
I reverted the changes!